libcodechecker/analyze/log_parser.py

```diff
@@ -112,6 +111,9 @@ def get_compiler_target(compiler):
 
 def parse_compile_commands_json(logfile, add_compiler_defaults=False):
```

Comment:
I would prefer to change the default argument to true and modify the callers accordingly.